controller/tap/apiserver.go

```diff
@@ -95,8 +95,9 @@ func (a *apiServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		for _, cn := range a.allowedNames {
 			for _, clientCert := range req.TLS.PeerCertificates {
 				clientNames = append(clientNames, clientCert.Subject.CommonName)
-				if cn == clientCert.Subject.CommonName {
-					validCN = clientCert.Subject.CommonName
+				// Check Common Name and Subject Alternate Name(s)
+				if cn == clientCert.Subject.CommonName || isSubjectAlternateName(clientCert, cn) {
+					validCN = cn
```

Comment:
I wonder why is validCN being a string while it seems to express a bool only information. Can we change that to a bool (i.e. validCNFound) or just restructure the function?

Reply:
I guess I didn't really give it much of a look beyond including my changes! 😄 Looking at it now however, there definitely seems to be no reason to treat `validCN` as a string.  I'll restructure the method to treat that as a boolean...thanks for the suggestion!